config/locales/en.yml

```diff
@@ -31,8 +31,8 @@ en:
   welcome_to: 'Welcome to '
   description: "We begin from the ground up. With farmers and growers ready to tell their stories proudly and truly. With distributors ready to connect people with products fairly and honestly. With buyers who believe that better weekly shopping decisions can…"
   search_by_name: Search by name or suburb...
-  producers: Aussie Producers
-  producers_join: Australian producers are now welcome to join the Open Food Network.
+  producers: Aussie Producers #FIXME
```

Comment:
Maybe someone was not happy with the language? "Aussie Producers" is Australia specific.